source/compatibility.tex

```diff
@@ -3734,10 +3734,9 @@
 Subclause \ref{csetjmp.syn} describes the changes.
 
 \rSec3[diff.offsetof]{Macro \tcode{offsetof(\placeholder{type}, \placeholder{member-designator})}}
-\indexlibraryglobal{offsetof}%
 
 \pnum
-The macro \tcode{offsetof}, defined in
+The macro \tcode{\libmacro{offsetof}}, defined in
```

Comment:
\libmacro should make immediately surrounding \tcode superfluous, if defined appropriately.

Please clean up.